src/psyclone/tests/parse/algorithm_test.py

```diff
@@ -288,7 +316,7 @@ def test_getkernel_noexpr(parser, content):
         in str(excinfo.value)
 
 
-def test_getkernel_argerror(monkeypatch, parser):
+def test_getkernel_argerror(monkeypatch):
     '''Test that the get_kernel function raises an exception if it does
     not recognise the fparser2 parse tree for an argument.
```

Comment:
pylint complains about useless suppression of 'unused-argument' below.

Reply:
Fixed complaints.